src/libraries/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/CanonicalXmlComment.cs

```diff
@@ -49,19 +49,19 @@ public void WriteHash(HashAlgorithm hash, DocPosition docPos, AncestralNamespace
             if (!IsInNodeSet || !IncludeComments)
                 return;
 
-            UTF8Encoding utf8 = new UTF8Encoding(false);
-            byte[] rgbData = utf8.GetBytes("(char) 10");
+            Encoding encoding = Utils.DefaultEncoding;
+            byte[] rgbData = encoding.GetBytes("(char) 10");
```

Comment:
```suggestion
            byte[] rgbData = "(char) 10"u8.ToArray();
```
?  You can do the same for all the other GetBytes calls that are taking a literal.